### PR TITLE
chore: Bump stackable-operator and other dependencies

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
# Description

Bump stackable-operator to 0.70.0, and other dependency updates

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [x] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

### CODE REVIEW

**Comment by NickLarsenNZ** [comment]
Nightly test result

```
=== NAME  kuttl
    harness.go:408: run tests finished
    harness.go:516: cleaning up
    harness.go:573: removing temp folder: ""
--- PASS: kuttl (49.67s)
    --- PASS: kuttl/harness (0.00s)
        
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Bump `stackable-operator` to 0.70.0 ([#267]).
```
Comment (xeniape) [inline CHANGELOG.md:9]:
```suggestion
- Bump `stackable-operator` to 0.70.0, and other dependencies ([#267]).
```
Not sure if product-config needs a separate mention, but if not it's incorporated with this as well.

**Review by xeniape — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (NickLarsenNZ) [reply CHANGELOG.md:9]: There is no `product-config` dependency in the commons operator (it's a bit different since it isn't a product specific operator).

Comment (xeniape) [reply CHANGELOG.md]: Ah, I saw it here and assumed https://github.com/stackabletech/commons-operator/pull/267/files#diff-ac358776bc52d3fe028a6aece6b81a472e30226748e420de1dca6de541028020L4 okay then! :smile:

**Review by xeniape — APPROVED** [review]

**Pull request merged**